### PR TITLE
[skip ci]add CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by wxyucs**
add a standard CODE_OF_CONDUCT.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+.
```
Comment (jiaweizone) [inline CODE_OF_CONDUCT.md]: here need add email ？

Comment (wxyucs) [reply CODE_OF_CONDUCT.md]: updated

**Review by LHT129 — APPROVED** [review]
LGTM

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**